packages/reporters/liquidation-reporter/emp-position-helper.ts

```diff
@@ -0,0 +1,32 @@
+const fetch = require("node-fetch");
+
+export async function getAllEmpsPositions() {
+  const subgraphUrl = "https://api.thegraph.com/subgraphs/name/umaprotocol/uma";
+  const query = `
+    query activePositions {
+      financialContracts {
+        id
+        positions(first: 1000, where: { collateral_gt: 0 }) {
```

Comment:
is 1000 too small? Why not set to something insanely high like 1000000

Reply:
what financial contract has that many positions? could change it if need be but right now we dont have even half that.